source/framework/tools/src/TRestTools.cxx

```diff
@@ -878,8 +880,13 @@ std::string TRestTools::POSTRequest(const std::string& url, const std::map<std::
     fclose(f);
     curl_global_cleanup();
 
-    std::string file_content = "";
     std::getline(std::ifstream(filename), file_content, '\0');
+#else
+    ferr << "TRestTools::POSTRequest. REST framework was compiled without CURL support" << endl;
```

Comment:
I think we should create a `RESTException` for these cases. Shall I create an issue about this?

Reply:
Yes, if you like, please, explain at the issue how do you want to handle it. For the moment we use no expections, as you see, as it is it just prompts a warning message, then does nothing. Not sure if in this particular case it requires of error handling.

Reply:
I would say wherever we have `exit()` in the code we should replace by a `throw` with an exception handler. Perhaps not for this particular case...